src/Charcoal/Admin/Widget/ObjectFormWidget.php

```diff
@@ -344,7 +370,8 @@ public function widgetDataForJs()
             'tab'                => $this->isTabbable(),
             'group_display_mode' => $this->groupDisplayMode(),
             'group_conditions'   => $this->groupsConditionalLogic(),
-            'allow_reload'       => $this->allowReload()
+            'allow_reload'       => $this->allowReload(),
+            'force_refresh'       => $this->forceRefresh()
```

Comment:
Fix alignment of `key => value` assignment operators and add trailing comma for multi-line array declarations.

```suggestion
            'allow_reload'       => $this->allowReload(),
            'force_refresh'      => $this->forceRefresh(),
```